src/main/java/com/ning/http/client/multipart/Part.java

```diff
@@ -12,13 +12,13 @@
  */
 package com.ning.http.client.multipart;
 
-import static java.nio.charset.StandardCharsets.US_ASCII;
-
 import java.io.IOException;
 import java.io.OutputStream;
 import java.nio.channels.WritableByteChannel;
 import java.nio.charset.Charset;
 
+import static java.nio.charset.StandardCharsets.US_ASCII;
+
```

Comment:
Could you please revert this? This change is unrelated to your fix and is probably due to different IDE settings.